src/components/Home.js

```diff
@@ -3,71 +3,28 @@ import { Link } from 'react-router';
 import CircularProgressbar from 'react-circular-progressbar';
 import userSettings from '../containers/userSettings';
 import userEvents from '../containers/userEvents';
-import userPushEvents from '../containers/userEvents';
 import Navigation from './Navigation';
 import helpers from './helpers';
 
 
 class Home extends React.Component {
   constructor() {
     super();
-    this.state = {
-      pushEvents: null,
-    };
 }
 
 goToRepos() {
   this.context.router.push('/repos');
 }
 
 grabUserInfo() {
-  fetch(`https://api.github.com/users/bcgodfrey91/events?page=0&callback`, {
+  fetch(`https://api.github.com/users/kylem038/events?page=0&callback`, {
     method: 'GET'
   })
   .then((res) => {return res.json(); })
```

Comment:
We definitely can, I just set it up this way as to not confuse the group by using two different instances of `res` (given that they're returning different values)